### PR TITLE
Offer links to CSP alternatives?

### PR DESCRIPTION
**Pull request by stilliard**
Added links to a couple tools that help in finding mixed content when CSP is not viable.

### CODE REVIEW

File: src/content/en/fundamentals/security/prevent-mixed-content/fixing-mixed-content.md [hunk 1]
```diff
@@ -298,3 +298,9 @@ The downside of using `block-all-mixed-content` is, perhaps obviously, that all
 content is blocked. This is a security improvement, but it means that these 
 resources are no longer available on the page. This might break features and 
 content that your users expect to be available. 
+
+### Alternatives to CSP
+
+If your site is hosted for you by a platform such as Blogger, you may not have access to modify headers & add a CSP. 
+Instead a viable alternative could be to use a website crawler to find issues across your site for you, such as 
+[HTTPSChecker](https://httpschecker.net/how-it-works#httpsChecker){: .external } or [Mixed Content Scan CLI](https://github.com/bramus/mixed-content-scan){: .external }
```
Comment (petele) [inline src/content/en/fundamentals/security/prevent-mixed-content/fixing-mixed-content.md]: Please split lines at 80 characters, so break the line at the start of `[Mixed Content...`

**Comment by petele** [comment]
got an LGTM by email from @ericlaw1979

**Comment by stilliard** [comment]
Thanks @petele & @ericlaw1979, I've now split the lines to 80 characters. Hope this is ok.

**Comment by petele** [comment]
Looks good - will push on Monday.

**Pull request merged**